assets/src/edit-story/components/header/buttons.js

```diff
@@ -147,7 +150,7 @@ function PreviewButton() {
 
   return (
     <>
-      <Outline onClick={openPreviewLink} isDisabled={isSaving}>
+      <Outline onClick={openPreviewLink} isDisabled={isSaving || isUploading}>
```

Comment:
Do we need to do the same with "Save draft" button?

Reply:
We already do that:

https://github.com/google/web-stories-wp/blob/7025fab42f299f2b30f3e9b06e4069396e02e997/assets/src/edit-story/components/header/buttons.js#L250-L257